packages/zoe/test/unitTests/test-instanceStorage.js

```diff
@@ -70,7 +73,8 @@ test('makeAndStoreInstanceRecord', async t => {
   // Add currency again, but call it "money"
   addIssuerToInstanceRecord(
     'Money',
-    makeIssuerRecord(currencyKit.brand, currencyKit.issuer, AssetKind.NAT, {
+    makeIssuerRecord(currencyKit.brand, currencyKit.issuer, {
+      assetKind: AssetKind.NAT,
```

Comment:
I think the brace moved.  It looks like the signature for `makeIsuserRecord()` is still
```
 * @param {Brand} brand
 * @param {Issuer} issuer
 * @param {DisplayInfo=} displayInfo
 * @returns {IssuerRecord}
```